src/legacy/core_plugins/kibana/public/kibana.js

```diff
@@ -69,4 +69,11 @@ routes
     redirectTo: `/${chrome.getInjected('kbnDefaultAppId', 'discover')}`
   });
 
-uiModules.get('kibana').run(showAppRedirectNotification);
+uiModules.get('kibana', [
```

Comment:
I think the inclusion of these dependencies in the `kibana` module should be in a more central place as the `kibana` module is not just used by the kibana app, but this file is only loaded for the kibana app.

Maybe here: https://github.com/elastic/kibana/blob/168d17e4746d6932845eddf1e648375113bcf825/src/ui/public/chrome/api/angular.js#L38

Reply:
Solved accordingly your feedback and what we agreed in the zoom call 😃